topics/appendix/tools/gradle_intellij_plugin/tools_gradle_intellij_plugin.md

```diff
@@ -1638,3 +1638,19 @@ Type
 
 Default value
 : `${prepareSandboxTask.destinationDir}/${prepareSandboxTask.pluginName}`
+
+
+## verifyPluginConfiguration Task
+
+> Available since the upcoming release
+>
+{type="warning"}
+
+Validates the plugin project configuration:
+
+- The [`patchPluginXml.sinceBuild`](#patchpluginxml-task-sincebuild) property can't be lower than the major version of the currently used IntelliJ SDK set with the [`intellij.version`].
+- The `sourceCompatibility` property of the Java configuration can't be lower than the Java version used for assembling the IntelliJ SDK specified by the [`intellij.version`].
```

Comment:
what does "for assembling" mean? "is built with"?

Reply:
Yes.